news/2019-02-05-osu-mapping-olympiad-4-results.md

```diff
@@ -0,0 +1,50 @@
+| :-: | --- | --- | :-: | :-: | :-: | :-: | :-: | :-: | :-: |
+| \#1 | Sinnoh | Zealous Beaver | 27.5 | 26.5 | 26 | 29 | 13 | 13 |  **135** |
+| \#2 | Minato Yukina | Esteemed Dragon | 27 | 24 | 24 | 26 | 12 | 13 | **126** |
+| \#3 | Ascendance | Puzzled Crow | 23.5 | 23 | 22 | 23.5 | 14 | 11.5 | **117.5** |
+| \#4 | -Luminate | Mountainous Sheep | 24 | 21 | 22 | 22 | 9 | 12.5 | **110.5** |
+| \#5 | Rocma | Imaginative Pintail | 25 | 20 | 19.5 | 22 | 10.5 | 12 | **109** |
+| \#6 | Xetopia | Jittery Lemur | 23 | 23 | 23 | 23 | 1 | 12 | **105** |
+| \#7 | IDu5t | Urban Alligator | 23.5 | 20.5 | 18.5 | 21.5 | 7 | 11.5 | **102.5** |
+| \#8 | chickenbible | Spherical Chameleon | 21.5 | 18 | 16 | 16 | 0 | 8 | **79.5** |
+| \#9 | hoangnobitang | Dutiful Manatee | 6 | 3 | 2 | 3 | 1 | 3 | **18** |
+| \#10 | Oeti | Aromatic Owl | 6 | 3 | 2 | 3 | 0 | 3 | **17** |
+
+Securing **first place** for the inaugural osu!catch Mapping Olympiad is [**Sinnoh**](https://osu.ppy.sh/users/4236057) with a total score of **135**! Sinnoh's entry blew away the competition, securing the highest aggregate scores in Musical Relevance, Creativity and Technique, and also garnered a perfect score from two of the three judges' in the Impression category.
+
+A strong **second place** is awarded to [**Minato Yukina**](https://osu.ppy.sh/users/531253) with a total score of **126**! Minato Yukina's entry tied with Sinnoh's for the best Difficulty Spread, and whilst not placing first in any particular category, put in a strong performance across the board to secure a respectably high finish from the veteran.
+
+Rounding out the podium in **third place** is [**Ascendance**](https://osu.ppy.sh/users/2931883) with a total score of **117.5**! Though he could not repeat his feats from 2017's osu!catch Beatmapping Contest, receiving the highest score for hitsounding and solid marks across all other categories helped the most recent *Elite Mapper* in osu!catch edge ahead of fierce competition.
+
+[Complete results for this content, including the judges' comments for each submission can be found here.](https://docs.google.com/spreadsheets/d/1QTQGLqWne1WeJIu36_lS_AIZ0et4-DJ-OngAVCftA08/edit?usp=sharing)
+
+A huge thank you to everyone who submitted an entry and congratulations to the top three finishers! Their rewards will be allocated as follows (as a reminder, 6 Olympiad Points are needed for one badge level):
+
+- **First place**: 4 months of osu!supporter, 6 Olympiad Points (resulting in 'Mapping Olympian' badge) and the *Elite Mapper* title
+- **Second place**: 3 months of osu!supporter, 4 Olympiad Points
+- **Third place**: 2 months of osu!supporter, 2 Olympiad Points
+
+[A complete listing of all accumulated Olympiad Points can be located here](https://docs.google.com/spreadsheets/d/1_gIDJwTOgIhGr2h4069-r1C-2GHuV--5wtwPywzYsz8/edit?usp=sharing) which will be updated shortly after each contest.
```

Comment:
```suggestion
[A complete listing of all accumulated Olympiad Points can be located here](https://docs.google.com/spreadsheets/d/1_gIDJwTOgIhGr2h4069-r1C-2GHuV--5wtwPywzYsz8/edit?usp=sharing), and it will be updated shortly after each contest.
```

Reply:
Decided to split these clauses into new sentences instead